kernel/examples/read-table-multi-threaded/src/main.rs

```diff
@@ -81,6 +81,7 @@ fn main() -> ExitCode {
 struct ScanFile {
```

Comment:
We don't need the `ScanFile` type defined here anymore. Since the kernel now has a similar struct, just use that one.

Same comment for all the other custom `ScanFile` defs.

Reply:
Yeah makes sense, looks pretty clean now imo